extensions/wikia/Recirculation/RecirculationApiController.class.php

```diff
@@ -48,48 +48,39 @@ public function getFandomPosts() {
 	}
 
 	public function getCakeRelatedContent() {
-		$this->cors->setHeaders($this->response);
+		$this->cors->setHeaders( $this->response );
 
-		$target = trim($this->request->getVal('relatedTo'));
-		if (empty($target)) {
-			throw new InvalidParameterApiException('relatedTo');
+		$target = trim( $this->request->getVal( 'relatedTo' ) );
+		if ( empty( $target ) ) {
+			throw new InvalidParameterApiException( 'relatedTo' );
 		}
 
-		$limit = trim($this->request->getVal('limit'));
-		$ignore = trim($this->request->getVal('ignore'));
+		$limit = trim( $this->request->getVal( 'limit' ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "limit" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)